src/nvim/api/private/helpers.h

```diff
@@ -29,6 +29,10 @@
   .data.integer = i                                                           \
   })
 
+#define FLOATING_OBJ(f) ((Object) { \
```

Comment:
Is this named `FLOATING_OBJ` because of member `floating`? Why not just call it `FLOAT_OBJ` for it's type?